app.js

```diff
@@ -974,9 +975,16 @@ define(function(require) {
 			if (data.isDevices) {
 				// remove extra data not parsed properly  .. Only Check this field for devices if they exist
 				_.each(formattedData.data.recordsToReview, function(record) {
-					record.brand = record.brand ? record.brand.toLowerCase() : '';
-					record.family = record.family ? record.family.toLowerCase() : '';
-					record.model = record.model ? record.model.toLowerCase() : '';
+					if (record.brand) {
+						record.brand.toLowerCase();
+					}
+					if (record.family) {
+						record.family.toLowerCase();
+					}
+					if (record.model) {
+						record.model.toLowerCase();
```

Comment:
`String#toLowerCase` is pure, as it does not mutate the original string, and returns a new one representing the calling string converted to lower case.

So in this case, those operations do not have any effect.